src/entity-factory/OldEntityFactory.ts

```diff
@@ -0,0 +1,14 @@
+import { EntityFactoryInterface } from './EntityFactoryInterface';
+
+export class OldEntityFactory implements EntityFactoryInterface {
```

Comment:
Alternatively we could just point code example in the changelog how user can do same without this being in the codebase. Im okay with OldEntityFactory name btw since it would be very temporary.